src/app/clusters/account-login-server/account-login-server.cpp

```diff
@@ -41,12 +42,15 @@ using namespace chip::AppPlatform;
 #endif // CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
 using chip::app::Clusters::AccountLogin::Delegate;
 
+#define ACCOUNT_LOGIN_DELEGATE_TABLE_SIZE                                                                                          \
```

Comment:
since C++, how about using `static constexpr size_t kAccountLogingDelegateTableSize = ...` ?

same below. I would preffer C++ constants to  c macros.

Reply:
This is nitpicky. Don't want to hold up PR on this. resolving, but I would appreciate a followup that fixes this.